### DISCUSSION TITLE
Cannot upload files

### DISCUSSION THREAD

**Discussion by onlinehuazai** [community/community#33591] [Other Feature Feedback, Questions, & Ideas]
Cannot upload files, 404 error

**Comment by lxvx** [comment]
Try putting your files in their names. Ex. python:   .py    Ex.    C++:     .cpp

**Reply by onlinehuazai** [reply]
> Try putting your files in their names. Ex. python: .py Ex. C++: .cpp

It does not work, I try it again and again

**Reply by lxvx** [reply]
This is weird- ive never heard / seen this glitch before...

**Comment by tusharhero** [comment]
can you try using git to push the files?
